src/pages/index.js

```diff
@@ -1,91 +1,94 @@
 import React from 'react'
 import { Link } from 'gatsby'
-
 import Layout from '../components/layout'
-import Image from '../components/image'
+import styles from './index.module.scss'
+import Image1 from '../images/team-illustration.svg'
```

Comment:
it might be easier to parse the Images inside code if the names were more descriptive. e.g: `Image1` -> `TeamIllustrationImage`